persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_36/generated/rainier/persist_client.bal

```diff
@@ -0,0 +1,282 @@
+// AUTO-GENERATED FILE. DO NOT MODIFY.
+
+// This file is an auto-generated file by Ballerina persistence layer for model.
+// It should not be modified by hand.
+
+import ballerina/persist;
+import ballerina/jballerina.java;
+import ballerinax/mysql;
+
+const 'BUILDING = "buildings";
```

Comment:
The persistClient map key contains `'`, But this constant value doesn't have  `'`. It may produce a runtime error by this https://github.com/ballerina-platform/persist-tools/pull/169/files#diff-5471498125c88e75cb7eee2d58951b6b206b9e57cc1e6891e96b17d05b5e46a6R162. Isn't it?
```suggestion
const 'BUILDING = "'buildings";
```

Reply:
updated in [350ab74](https://github.com/ballerina-platform/persist-tools/pull/169/commits/350ab7424902108e6b012b4b9649a5f17880dfa2)